.github/workflows/apisec-scan.yml

```
@@ -71,6 +71,8 @@ on:
 permissions:
   contents: read
 
+jobs:
```

actionlint (1.7.10)

[error] "jobs" section should not be empty. please remove this section if it's unnecessary

(syntax-check)